### ISSUE TITLE
Diagram of the network architecture

### ISSUE THREAD

**Issue by armytricks** [uzh-rpg/agile_autonomy#31]
Is there a diagram of the neural network architecture (specifically the branches and heads) used in this paper?

**Comment by antonilo** [comment]
Yes, we have one in the paper in Fig. 6. (https://rpg.ifi.uzh.ch/docs/Loquercio21_Science.pdf)
